requirements/small-requirements.txt

```diff
@@ -16,3 +16,4 @@ databricks-cli@git+https://github.com/databricks/databricks-cli.git
 pillow
 matplotlib
 shap>=0.40
+virtualenv
```

Comment:
Should `virtualenv` be included in mlflow's dependencies? I'm leaning towards no because virtualenv support is **still** experimental and optional.